node/src/reactor/joiner.rs

```diff
@@ -491,13 +491,6 @@ impl reactor::Reactor for Reactor {
                     });
                     self.dispatch_event(effect_builder, rng, event)
                 }
-                // needed so that consensus can notify us of the eras it knows of
```

Comment:
I don't really understand that comment anymore. But if you tested it, it should be fine. :+1:

Reply:
That seems to be related to an early solution of mine, which we eventually got rid of :sweat_smile: But yeah, the nodes seem to be doing just fine without this fragment.